docs/guides/explorer/configuring-pi-chains.mdx

```diff
@@ -2,23 +2,19 @@
 
 Hyperlane can be [permissionlessly deployed](/docs/deploy-hyperlane.mdx) to any chain, but messages on PI chains cannot be identified by the default Hyperlane agents. To view details about messages from PI chains, first configure the explorer with metadata about that chain.
 
-To begin, go to the [explorer's settings page](https://explorer.hyperlane.xyz/settings).
+To begin, click the either the Origin or the Destination button on the explorer's home page. This will open the chain selection modal. From there, click the plus icon in the top-right.
```

Comment:
```suggestion
To begin, click either the Origin or the Destination button on the explorer's home page. This will open the chain selection modal. From there, click the plus icon in the top-right.
```